src/components/GlobalLeaderboards/Context.tsx

```diff
@@ -0,0 +1,213 @@
+import { BigNumber, utils } from "ethers";
+import {
+  createContext,
+  FC,
+  PropsWithChildren,
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from "react";
+
+import { useChainId } from "lib/chains";
+import {
+  PerfPeriod,
+  LeaderboardContextType,
+  useTopAccounts,
+  useTopPositions,
+  TopAccountsRow,
+  TopPositionsRow,
+  Profiler,
+} from "domain/synthetics/leaderboards";
+
+export const LeaderboardContext = createContext<LeaderboardContextType>({
+  chainId: 0,
```

Comment:
we have simple useChainId hook, probably we don't need to store chainId in the context